src/AclExtras.php

```diff
@@ -294,7 +294,11 @@ protected function _checkMethods($className, $controllerName, $node, $pluginPath
         $excludes = $this->_getCallbacks($className, $pluginPath);
         $baseMethods = get_class_methods(new Controller);
         $namespace = $this->get_namesapce($className, $pluginPath);
-        $actions = get_class_methods(new $namespace);
+        $reflectionClass = new \ReflectionClass( $namespace );
+        $actions = null;
+        if ($reflectionClass->IsInstantiable()) {
+            $actions = get_class_methods(new $namespace);
```

Comment:
This is probably a bad idea given that the class may have required constructor parameters. It might be better to use the reflection instance to get the methods.